### ISSUE TITLE
pip install error

### ISSUE THREAD

**Issue by CBtiger** [eoyilmaz/displaycal-py3#156]
I am trying the pip install on a Ubuntu 22.04 (correction not 18.04) box. Its using a Python3.6.9 virtualenv. I get an error during the install process. 

Here is the output. I wonder that no other reported issues, so probably I did something wrong. I noted that the download is a tar.gz whereas other packages download as wheel file.    

```
pip install DisplayCAL
Collecting DisplayCAL
  Downloading DisplayCAL-3.9.7.tar.gz (10.7 MB)
     |████████████████████████████████| 10.7 MB 85 kB/s 
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  ERROR: Command errored out with exit status 1:
   command: /home/test/.pyenv/versions/3.6.9/envs/default/bin/python3.6 /home/test/.pyenv/versions/3.6.9/envs/default/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmp4ge1hoq3
       cwd: /tmp/pip-install-ejcjujso/DisplayCAL
  Complete output (22 lines):
  ['egg_info']
  Traceback (most recent call last):
    File "/home/test/.pyenv/versions/3.6.9/envs/default/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py", line 280, in <module>
      main()
    File "/home/test/.pyenv/versions/3.6.9/envs/default/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py", line 263, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/home/test/.pyenv/versions/3.6.9/envs/default/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py", line 114, in get_requires_for_build_wheel
      return hook(config_settings)
    File "/tmp/pip-build-env-6abgfqpa/overlay/lib/python3.6/site-packages/setuptools/build_meta.py", line 163, in get_requires_for_build_wheel
      config_settings, requirements=['wheel'])
    File "/tmp/pip-build-env-6abgfqpa/overlay/lib/python3.6/site-packages/setuptools/build_meta.py", line 143, in _get_build_requires
      self.run_setup()
    File "/tmp/pip-build-env-6abgfqpa/overlay/lib/python3.6/site-packages/setuptools/build_meta.py", line 158, in run_setup
      exec(compile(code, __file__, 'exec'), locals())
    File "setup.py", line 1583, in <module>
      setup()
    File "setup.py", line 991, in setup
      from DisplayCAL.setup import setup
    File "/tmp/pip-install-ejcjujso/DisplayCAL/DisplayCAL/setup.py", line 525
      if skip_instrument_conf_files := "--skip-instrument-configuration-files" in sys.argv[1:]:
                                     ^
  SyntaxError: invalid syntax
  ----------------------------------------
ERROR: Command errored out with exit status 1: /home/test/.pyenv/versions/3.6.9/envs/default/bin/python3.6 /home/test/.pyenv/versions/3.6.9/envs/default/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmp4ge1hoq3 Check the logs for full command output.
```

**Comment by p5k369** [comment]
Hey @CBtiger DisplayCal is compatible with python 3.7 up to 3.10.
It is not compatible with python 3.6.9. If xou want to make it run by yourself it this is the only section that fails than replace:
`if skip_instrument_conf_files := "--skip-instrument-configuration-files" in sys.argv[1:]:`
by
`skip_instrument_conf_files = "--skip-instrument-configuration-files" in sys.argv[1:]
if skip_instrument_conf_files:`

**Comment by CBtiger** [comment]
Thanks for your info, that gave me the missing hint. 
I had a type above, I got the error when moving from 18.04 -> 22.04. I had been using Python 3.6.9 in 18.04 with pyenv without problems.  

I tried 3.8.2 and it works. So this incompatibility must have come with with Ubuntu 22.04 repository dependencies. Its fine to move to 3.8.2 now, but I will note down your fallback for sure!

**Comment by p5k369** [comment]
Resolved by updating python version to at least 3.8
